server/server.go

```diff
@@ -8,11 +8,11 @@ import (
 	"github.com/milehighgophers/website/ui"
 )
 
-type Store interface {
-	AllEvents() *data.MeetupSchedule
-}
+// type Store interface {
+// 	AllEvents() *data.MeetupSchedule
+// }
 
-func Start(addr string, s Store) error {
+func Start(addr string, s *data.Store) error {
```

Comment:
I'm wondering why `*data.Store` was chosen over `Store`. Do you need the concrete type?

Reply:
I didn't see the advantage of an interface in this instance -- what's the best practice here?

Reply:
Typically, I receive small interfaces. This decouples your code, allows for callers to provide alternative implementations which is good for testing.

That said, passing concrete types is more performant. I only cross that bridge when it is necessary tho.